api/api.yaml

```diff
@@ -59,7 +59,17 @@ paths:
         - Facets
   /search:
     get:
-      description: Free text search over dataset
+      description: >
```

Comment:
I'm not able to get this to work. I did `git pull` on this branch. For 1000 Genomes, `http://localhost:4400/api/search?query=Puerto` should return results, but I don't get any results

Reply:
Hmm, not sure, because it works on my laptop/desktop. I deleted my local branch and checked out this branch from remote and tried. Did you get an errors?